layouts/partials/header.html

```diff
@@ -24,7 +24,7 @@ <h1 class="pull-left">
     {{ if eq .hero "homepage" }}
     <div class="hero hero-home">
       <div class="hero-text">
-        <h2 class="headline-main">IPFS is the Distributed Web</h2>
+        <h2 class="headline-main">IPFS (brings|enables|builds|for) the Distributed Web</h2>
```

Comment:
@autonome I'm voting for "enables".
@ericronne -- do you have an opinion?
Also @cwaring if he feels like chiming in from his holiday (but don't if you don't!)